tests/contrib/trace/conftest.py

```diff
@@ -18,6 +19,27 @@ def iopipe_with_trace_no_auto_measure():
     return IOpipe(token='test-suite', url='https://metrics-api.iopipe.com', debug=True, plugins=[plugin])
 
 
+@pytest.fixture
+def iopipe_with_trace_auto_http():
+    plugin = TracePlugin(auto_http=True)
+    return IOpipe(token='test-suite', url='https://metrics-api.iopipe.com', debug=True, plugins=[plugin])
+
+
+@pytest.fixture
+def iopipe_with_trace_auto_https():
+    plugin = TracePlugin(auto_http=True)
+    return IOpipe(token='test-suite', url='https://metrics-api.iopipe.com', debug=True, plugins=[plugin])
+
+
+@pytest.fixture
+def iopipe_with_trace_auto_http_filter():
+    def http_filter(response):
+        return not response.request.url.startswith('https://www.iopipe.com')
```

Comment:
Huh, metrics-api.iopipe.com doesn't "start with" www.iopipe com, so I'm not clear why this works (feels like it shouldn't?)

Reply:
Because this isn't the fixture being tested. The handler is what is being tested. See the `requests.get(...)` in the relevant handlers.

Reply:
Got it, thank you!